src/main/java/com/google/swarm/tokenization/common/BigQueryReadTransform.java

```diff
@@ -64,23 +66,28 @@ public PCollection<KV<String, TableRow>> expand(PBegin input) {
 
     switch (readMethod()) {
       case DEFAULT:
-        return input
-            .apply(
-                "ReadFromBigQuery",
-                BigQueryIO.readTableRows()
-                    .fromQuery(query())
-                    .usingStandardSql()
-                    .withMethod(Method.DEFAULT))
-            .apply("AddTableNameAsKey", WithKeys.of(tableRef()));
+        if(query()!=null) {
+          return input
+                  .apply(
+                          "ReadFromBigQuery",
+                          BigQueryIO.readTableRows()
+                                  .fromQuery(query())
+                                  .usingStandardSql()
+                                  .withMethod(Method.DEFAULT))
+                  .apply("AddTableNameAsKey", WithKeys.of(tableRef()));
+        }
+        else {
+          return input
+                  .apply(
+                          "ReadFromBigQuery",
+                          BigQueryIO.readTableRows()
+                                  .from(tableRef())
+                                  .withMethod(Method.DEFAULT))
+                  .apply("AddTableNameAsKey", WithKeys.of(tableRef()));
+        }
+
       case EXPORT:
-        return input
-            .apply(
-                "ReadFromBigQuery",
-                BigQueryIO.readTableRows()
-                    .fromQuery(query())
-                    .usingStandardSql()
-                    .withMethod(Method.DEFAULT))
-            .apply("AddTableNameAsKey", WithKeys.of(tableRef()));
+        throw new IllegalArgumentException("Export method not supported");
```

Comment:
May not be the intended behaviour but since the EXPORT case used to work with DEFAULT read earlier, are we declaring that somewhere before removing the support altogether?